CMSgov/design-system
# [NO-TICKET] Fix alt icon color on hover for filter chip

The color was not being changed on the alternate icon hover of the filter chip. 
This was because the icon had an inline fill attribute that was not getting overridden.

The fix was to remove the fill from the SVG and let it be defined in the SCSS like the other version of the close icon

### How to test
Demo 
http://design-system-demo.s3-website-us-east-1.amazonaws.com/no-ticket-filter-chip-hover-fix/ 

Go to the filter chip page and hover over the version which uses the alternate close icon and see that the color of the close icon changes from a dark grey to white on hover.
